html-and-web/html-and-the-web_personal-website/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+
+    <body style="background-color:rgb(255, 165, 127);"></body>
```

Comment:
nice touch; in the future this should go into a css stylesheet, not directly in the HTML-file

Reply:
noted :)